ui/src/components/entity/SimpleSymbolGloss.tsx

```diff
@@ -168,23 +169,25 @@ class SimpleSymbolGloss extends React.PureComponent<Props, State> {
               )}
               {this.props.showDrawerActions && (
                 <React.Fragment>
-                  <td>
-                    <MuiTooltip
-                      title={
-                        defsAndNicknames.length > 0
-                          ? `See ${defsAndNicknames.length} definitions`
-                          : "No definitions."
-                      }
-                    >
-                      <IconButton
-                        size="small"
-                        disabled={defsAndNicknames.length === 0}
-                        onClick={this.onClickDefinitionsButton}
+                  {definitionsEnabled && (
+                    <td>
+                      <MuiTooltip
+                        title={
+                          defsAndNicknames.length > 0
+                            ? `See ${defsAndNicknames.length} definitions`
```

Comment:
> See 1 definitions

Hmmm...

> See 1000 definitions

...ummmm

```suggestion
                            ? `See ${defsAndNicknames.length.toLocaleString()} definition${defsAndNicknames.length === 1 ? '' : 's'}`
```